### PR TITLE
Deniz Uzun - Lab3

### PR DESCRIPTION
**Pull request by denizeu**

### CODE REVIEW

File: src/needleman_wunch.jl [hunk 1]
```diff
@@ -5,10 +5,13 @@
...
-function nwscore(base1::Char, base2::Char)
-    
-end
-
+function nwscore(base1::Char, base2::Char; match=1, mismatch=-1)
+        if base1 == base2
+            return 1
+        end
+        if base1 != base2
+            return -1
```
Comment (kescobo) [inline src/needleman_wunch.jl]: Looks like and end is missing

**Review by kescobo — CHANGES_REQUESTED** [review]

**Comment by denizeu** [comment]
Do you have any advice on how to add the match and mismatch within the code? Did I define it properly in the function? @kescobo

File: src/needleman_wunch.jl [hunk 2]
```diff
@@ -5,11 +5,14 @@
 
 Your documentation here.
 """
-function nwscore(base1::Char, base2::Char)
-    
-end
-
+function nwscore(base1::Char, base2::Char; match, mismatch)
```
Comment (kescobo) [inline src/needleman_wunch.jl]: The purpose of `match` and `mismatch` here is to provide an alternative scoring system. So I think you want something like

```
function wscore(base1::Char, base2::Char; match=1, mismatch=-1)
    if base1 == base2
        return match
    end
    # ... etc
end
```

**Review by kescobo — CHANGES_REQUESTED** [review]
I see you've been trying a lot of stuff, but may be going around in circles a bit. Let me know if we can hop on a call today or tomorrow.

**Comment by denizeu** [comment]
> I see you've been trying a lot of stuff, but may be going around in circles a bit. Let me know if we can hop on a call today or tomorrow.

A zoom call tomorrow would be super helpful, thank you! @kescobo

**Comment by kescobo** [comment]
Not sure why the automated tests are failing to run here - let me know if you think you're done and I'll run them locally

**Comment by denizeu** [comment]
@kescobo I am definitely done with lab3 but not with lab4... it might be because I am continuing to work on lab4 through this same file? Anika and I decided that would be the best way to still have the old needleman_wunch to work on for lab4, maybe there is a better way?

**Comment by denizeu** [comment]
@kescobo I'm actually a bit more confused now. I did lab 6 with Emily which should be done, and I also finished up lab 3. I should only be missing/working on lab 4, I think the locations are unclear for me at the moment. This branch should be the needleman_wunch uncorrected file with only lab 3 and lab 4 right?

**Comment by kescobo** [comment]
Lab 3 complete as of https://github.com/wellesley-bisc195/AlignmentAlgorithms/pull/2/commits/3f4fe7fde2ed620d72c444dd71d8eb3a6a4f4069

**Pull request closed without merging**